pkg/reconciler/v1alpha1/clusteringress/config/store_test.go

```diff
@@ -30,28 +31,41 @@ func TestStoreLoadWithContext(t *testing.T) {
 	store := NewStore(TestLogger(t))
 
 	istioConfig := ConfigMapFromTestFile(t, IstioConfigName)
+	networkConfig := ConfigMapFromTestFile(t, network.ConfigName)
 	store.OnConfigChanged(istioConfig)
+	store.OnConfigChanged(networkConfig)
 	config := FromContext(store.ToContext(context.Background()))
 
-	expected, _ := NewIstioFromConfigMap(istioConfig)
-	if diff := cmp.Diff(expected, config.Istio); diff != "" {
+	expectedIstio, _ := NewIstioFromConfigMap(istioConfig)
```

Comment:
do you want to verify that `ctx` carries the config?

Reply:
Added the check for the `error` from the `NewIstioFromConfigMap` function.